lib/puppet/type/acl.rb

```diff
@@ -184,7 +195,7 @@ def insync?(is)
         s = p.tr '-', ''
         r << (s.sub!('r', '')?'r':'-')
         r << (s.sub!('w', '')?'w':'-')
-        r << (s.sub!('x', '')?'x':'-')
+        r << (s.sub!('x', '')?'x':'-')#'
```

Comment:
Does `?'` have special meaning in ruby? That seems to be where github and emacs are freaking out, in L196